opensrp-chw-anc/src/main/java/org/smartregister/chw/anc/util/JsonFormUtils.java

```diff
@@ -218,6 +223,30 @@ public static String getValue(JSONObject jsonObject, String key) {
         return "";
     }
 
+    /**
+     * Returns an Array value from JSON form field
+     *
+     * @param jsonObject native forms jsonObject
+     * @param key        field object key
+     * @return Array value
+     */
+    public static JSONArray getValueArray(JSONObject jsonObject, String key) {
+        try {
+            JSONArray jsonArray = jsonObject.getJSONObject(JsonFormConstants.STEP1).getJSONArray(JsonFormConstants.FIELDS);
+            int x = 0;
+            while (jsonArray.length() > x) {
+                JSONObject jo = jsonArray.getJSONObject(x);
```

Comment:
Could you rename `jo` to something more descriptive?

Reply:
Refactored ✅